exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/__init__.py

```diff
@@ -69,3 +69,7 @@
 API
 ---
 """
+from .version import __version__
+
+otel_string = "OTel OTLP Exporter Python/" + __version__
+_OTLP_GRPC_HEADERS = [("user-agent", otel_string)]
```

Comment:
This name could be more descriptive of what it contains. And it can't be a public symbol since it's for internal purposes only.

```suggestion
_user_agent_header_value = "OTel OTLP Exporter Python/" + __version__
_OTLP_GRPC_HEADERS = [("user-agent", _user_agent_header_value)]
```

Reply:
Hi @srikanthccv, Could you re-add this suggestion? I didn't know we had a commit suggestion option before. Now, it won't let me commit to this. Thanks in advance.